Fluent/Post_Porcess.ipynb

```diff
@@ -65,14 +109,14 @@
     "\n",
     "At this stage one can conclude that this CFD simulation is validated to study unsteady laminar flow over a rotating cylinder. Therefore, these simulations can be used for similar or more complex problems, such as three dimensional flow over a rotating cylinder/sphere, flow around a bridge pillars, flow around supprting structures of an onshore oil well and many more similar problems. The user has to vary the mesh and flow conditions accordingly. The basic methodology for analysis remains the same. \n",
     "\n",
-    "Complete report of the simulation can be found [here](https://www.researchgate.net/publication/288183448_NUMERICAL_INVESTIGATION_OF_UNSTEADY_FLOW_ROUND_A_CIRCULAR_CYLINDER). A couple of YouTube videos are available depicting the vortex shedding and disappearance of vortices at high $\\alpha$ [here](https://www.youtube.com/channel/UCnYGMsr9wYHanrNxKOIEEDQ).\n",
     "\n",
     "References:\n",
-    "1. Kang, S., 2006, \"Laminar Flow over a Steadily Rotating Circular Cylinder under the influence of Uniform Shear\", Physics of Fluid, Vol. 18, No. 4, pp.47-106\n",
-    "2. Mittal, S., and Kumar, B., 2003, Flow Past a Rotating Cylinder\", Journal of Fluid Mech., Vol.476, pp.303-334.\n",
-    "3. Badr, H.M., Dennis, S.C.R., and Young, P.J.S., 1989, \"Steady and Unsteady Flow past a Rotating Circular Cylinder at Low Reynolds Numbers\", Computers and Fluids, VOl.17, No.4, pp.579-609.\n",
-    "4. [\"A Physical Introduction to Fluid Mechanics\"](http://www.efluids.com/efluids/books/Smits_text_part1.pdf), Alexander J Smits, 2014.\n",
-    "5. Cornell Fluent Learning Modules - [Unsteady Flow Past a Circular Cylinder](https://confluence.cornell.edu/display/SIMULATION/FLUENT+-+Unsteady+Flow+Past+a+Cylinder).\n"
+    "1. Cite your report [link](https://www.researchgate.net/publication/288183448_NUMERICAL_INVESTIGATION_OF_UNSTEADY_FLOW_ROUND_A_CIRCULAR_CYLINDER)\n",
+    "2. Kang, S., 2006, \"Laminar Flow over a Steadily Rotating Circular Cylinder under the influence of Uniform Shear\", Physics of Fluid, Vol. 18, No. 4, pp.47-106\n",
+    "3. Mittal, S., and Kumar, B., 2003, Flow Past a Rotating Cylinder\", Journal of Fluid Mech., Vol.476, pp.303-334.\n",
+    "4. Badr, H.M., Dennis, S.C.R., and Young, P.J.S., 1989, \"Steady and Unsteady Flow past a Rotating Circular Cylinder at Low Reynolds Numbers\", Computers and Fluids, VOl.17, No.4, pp.579-609.\n",
+    "5. [\"A Physical Introduction to Fluid Mechanics\"](http://www.efluids.com/efluids/books/Smits_text_part1.pdf), Alexander J Smits, 2014.\n",
+    "6. Cornell Fluent Learning Modules - [Unsteady Flow Past a Circular Cylinder](https://confluence.cornell.edu/display/SIMULATION/FLUENT+-+Unsteady+Flow+Past+a+Cylinder)."
```

Comment:
Remove this citation. It does not add any value to the content of work, unless it does.